style.css

```diff
@@ -77,6 +77,15 @@ header {
   z-index: 999;
 }
 
+.modal .popup a {
+  display: inline-flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  align-items: center;
+  gap: 0.5rem;
+  justify-content: space-between;
+}
```

Comment:
- [ ] This is really close but when checking responsiveness for mobile devices the ideal screen width is around `375px` at this screen width the text in the buttons of the popup modal is overlapping so I kindly suggest you fix that to make your UI more polished.
- Your design:
  ![image](https://github.com/Porkalaikarthika/Portfolio/assets/104968236/5fa24e43-5c2d-414e-a76f-e81ae8a529e1)
- Figma design:
  ![image](https://github.com/Porkalaikarthika/Portfolio/assets/104968236/3538043c-f1d8-4ec6-a08b-0edfdfa01934)